ServerAPI/Game.cs

```diff
@@ -1,116 +1,68 @@
-﻿using System;
-using System.Linq;
-using System.Threading.Tasks;
+namespace ServerAP;
 
-namespace TicTacToe;
-public enum GameSigns
-{
-    Empty,
-    O,
-    X,
-         
-}
 public class Game
 {
     private readonly int[,] _field = new int[3, 3]
-    { // 0  1  2
+    {
+        // 0  1  2
         {7, 8, 9}, //0
         {4, 5, 6}, //1
-        {1, 2, 3}  //2
+        {1, 2, 3} //2
     };
-
+    
+    
+    public static List<Player> Players = new();
+    public TurnInfo TurnInfo { get; set; } = null!;
```

Comment:
why you need to store turn info in the game?

Reply:
Actually don't need.Thank you for your help and experience wich you give my in this beautiful way.I really appreciate it.